pkg/monitors/filesystems/partitions_windows.go

```diff
@@ -0,0 +1,123 @@
+// +build windows
+
+package filesystems
+
+import (
+	"strings"
+	"syscall"
+
+	gopsutil "github.com/shirou/gopsutil/disk"
+	"golang.org/x/sys/windows"
+)
+
+func (m *Monitor) getPartitions(all bool) ([]gopsutil.PartitionStat, error) {
```

Comment:
It'd be helpful to explain what this does and attribute https://github.com/shirou/gopsutil/pull/781 with the 3 bsd  clauses.

Reply:
Wait he opened that PR in 2019?! Why hasn't his PR merged? I wanted to submit a PR to break up PartitionsWithContext into getting mount points and getting partition stats using the list of mount points.

Reply:
This implementation is inspired more by kostix's answer in https://stackoverflow.com/questions/62731962/get-list-of-fixed-drives

Reply:
Added comments and attributions.